packages/gatsby-theme-newrelic/src/utils/localization.js

```diff
@@ -11,3 +11,16 @@ export const localizePath = ({ path, locale }) => {
     ? path
     : `/${locale.localizedPath}${addLeadingSlash(path)}`;
 };
+
+export const localizeExternalLink = ({ link, locale }) => {
+  if (locale.isDefault) {
+    return link;
+  }
+
+  const paths = link.replace('https://', '').split('/');
+
+  paths.splice(1, 0, locale.locale).join('/');
```

Comment:
Rather than using array methods, which get just a little hard to follow, you could clean this up a little bit by using the `URL` constructor. Doing so also allows you to take advantage of the `localizePath` helper above.

```js
const url = new URL(link)

url.pathname = localizePath({ path: url.pathname, locale })

return url.href
```

Reply:
oooh yes, I'm good with that! Let me make that refactor

Reply:
@jerelmiller It has been updated! tested it out with all the links and works great :)